tests/test_io.py

```diff
@@ -294,3 +294,66 @@ def test_load_neo_file_nwb():
             for trace in traces:
                 assert trace['stim_start'] == [250]
                 assert trace['stim_end'] == [600]
+
+
+@pytest.fixture
+def json_filename():
+    filename = 'tmp.json'
+    yield filename
+    if os.path.exists(filename):
+        os.remove(filename)
+
+
+@pytest.fixture
+def csv_filename():
+    filename = 'tmp.csv'
+    yield filename
+    if os.path.exists(filename):
+        os.remove(filename)
+
+
+def test_save_feature_to_json(json_filename):
+    """Test saving of the features to JSON file."""
+    import efel
+    import os
+    import json
+    feature_values = {
```

Comment:
Are we sure that this is exactly what efel would return? No numpy stuff?

Reply:
Also would be interesting to have some null/None values in it (not sure which one to sure exactly, use whatever efel is using)

Reply:
Good point, I have switched to using the output from `get_feature_values` instead and tested with some more traces that have some or all None feature values